src/components/icon/__snapshots__/icon.test.jsx.snap

```diff
@@ -3,7 +3,6 @@
 exports[`Icon has defaults 1`] = `
 <svg
   className="icon-base"
-  focusable={false}
```

Comment:
So I read into this...

This _is visually supported_ in Edge. It is also supported in IE (but missing styles). I think perhaps it is also supported in Chrome / Firefox / Safari but it might be the same thing as IE (basically missing styles).

I don't have a use-case for now. But it was worthwhile to explore the focusability problem of SVGs for a bit. 

We might never have to handle this if we wrap focusable SVGs in a <Button /> or something. 

Just spreading context between us.